src/utils/getter.ts

```diff
@@ -0,0 +1,19 @@
+export function cacheObjectGetters<T, K extends PropertyKey = keyof T>(
```

Comment:
It seems that the second parameter is unnecessary, we can get all `get` properties by iterating the `this.info`. That way, we can make the calling of `cacheObjectGetters` more concise, and the above comments also no longer needed. Although this might incur a slight performance overhead, but I feel it's acceptable, considering `this.info` doesn't have many properties.

Reply:
Yeah I thought about doing that too, but there's one property `ast` which we don't want to cache. It already has its own LRU cache. Performance was something I feared too since there could be many module infos in the graph.

If you prefer, I can refactor this to handle the `ast` prop too, e.g. a `excludeKeys` param, or hardcode but rename the function to something specific. Either ways work for me depending on which you prefer 👍

Reply:
But the `ast` property is not a getter.

Reply:
It is set here: https://github.com/rollup/rollup/blob/6ead1f7ab4b4466de438d7b066b16b31d2c48340/src/Module.ts#L896

Reply:
Oh, I see it. the current resolution looks good to me now.